pkg/agent/controller/egress/egress_controller.go

```diff
@@ -443,6 +500,22 @@ func (c *EgressController) unbindPodEgress(pod, egress string) (string, bool) {
 	return "", false
 }
 
+func (c *EgressController) updateEgressStatus(egress *crdv1a2.Egress, nodeName string) error {
+	klog.V(2).Infof("Egress status : %#v, update to : %s", egress.Status, nodeName)
+	if egress.Status.NodeName == nodeName {
```

Comment:
Move to before the log?

Reply:
Logging here we can know the update event occured when the egress.Status.NodeName == nodeName

Reply:
got it, move the log 
```
	if egress.Status.NodeName == nodeName {
		return nil
	}
	klog.V(2).InfoS("Updating egress status", "egress", egress.Name, "oldNode", egress.Status.NodeName, "newNode", nodeName)
```